cli/resources/org/partiql/cli/syntax/PartiQL.nanorc

```diff
@@ -0,0 +1,34 @@
+syntax "PartiQL" "\.pql$"
+
+# Numbers
+color magenta "([0-9]+(\.)?[0-9]*|[0-9]*(\.)?[0-9]+)([eE][+-]?[0-9]+)*"
```

Comment:
I don't think this handles decimals specified with the `d`/`D`

<img width="388" alt="image" src="https://user-images.githubusercontent.com/23245405/174169698-577b535d-578a-4882-931d-92e9052adfc2.png">

Reply:
This next revision is better because it resolves this. I'll leave more info below. Already updated the PR description. Ion still isn't added for syntax, but now it's a much more straightforward path.